grobid-core/src/main/java/org/grobid/core/engines/HeaderParser.java

```diff
@@ -828,23 +807,14 @@ public BiblioItem resultExtraction(String result, List<LayoutToken> tokenization
                 else*/
                 if (biblio.getTitle() == null) {
                     biblio.setTitle(clusterContent);
-                    List<LayoutToken> tokens = getLayoutTokens(cluster);
-                    biblio.addTitleTokens(tokens);
                 }
             } else if (clusterLabel.equals(TaggingLabels.HEADER_AUTHOR)) {
                 //if (biblio.getAuthors() != null && isDifferentandNotIncludedContent(biblio.getAuthors(), clusterContent)) {
                 if (biblio.getAuthors() != null) {
                     biblio.setAuthors(biblio.getAuthors() + "\t" + clusterNonDehypenizedContent);
                     //biblio.addAuthorsToken(new LayoutToken("\n", TaggingLabels.HEADER_AUTHOR));
-                    biblio.addAuthorsToken(new LayoutToken("\t", TaggingLabels.HEADER_AUTHOR));
```

Comment:
Ok I realized that this is necessary to keep for properly segmenting the authors later in the process and attach the right affiliations. For further segmenting the authors, we need to keep track of discontinuous author fields and right now we inject a "\t" for this. Usually it means that an affiliation block appears between two authors. 

Injecting the `\t` is a bit hacky, we could think about something else not touching the local Layout token list for the author parser, or keep the "AuthorsTokens".

Reply:
I see, I put it back as it was. Trying to change name and add a comment.